test/test-exception.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// Testcase to produce NodeReport on uncaught exception
+if (process.argv[2] === 'child') {
+  require('../').setEvents('exception');
+
+  function myException(request, response) {
+    const m = '*** exception.js: testcase exception thrown from myException()';
+    throw new UserException(m);
+  }
+
+  function UserException(message) {
+    this.message = message;
+    this.name = 'UserException';
+  }
+
+  myException();
+} else {
+  const common = require('./common.js');
+  const spawn = require('child_process').spawn;
+  const tap = require('tap');
+
+  const child = spawn(process.execPath, [__filename, 'child']);
+  child.on('exit', (code, signal) => {
+    const expectedExitCode = process.platform === 'win32' ? 3221225477 : null;
```

Comment:
Just a suggestion, but `0xC0000005` is arguably more recognizable than `3221225477`.